hack/e2e-kueueviz-backend.sh

```diff
@@ -31,9 +31,14 @@ fi
 # Function to clean up background processes
 cleanup() {
   echo "Cleaning up kueueviz processes"
+<<<<<<< HEAD
   kill "${BACKEND_PID}" 
   cluster_cleanup "${KIND_CLUSTER_NAME}"
   restore_managers_image
+=======
+  kill "${BACKEND_PID}"
+  cluster_cleanup "${KIND_CLUSTER_NAME}"
+>>>>>>> 5dbc73b26 (<CARRY> create a script and onboard release 0.12)
```

Comment:
This file needs to be fixed. I also see this issue in the latest version of this branch
https://github.com/openshift-kannon92/kubernetes-sigs-kueue/blob/cherry-pick-openshift-changes/hack/e2e-kueueviz-backend.sh#L52

Reply:
fixed.